src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <span class="calendar__day calendar__day--number--1"></span>
+      <span class="calendar__day calendar__day--number--2"></span>
+      <span class="calendar__day calendar__day--number--3"></span>
+      <span class="calendar__day calendar__day--number--4"></span>
```

Comment:
I believe that you could complete this task without this modifiers for every day 🙂